prboom2/src/g_game.c

```diff
@@ -1586,6 +1586,12 @@ void G_DoCompleted (void)
 		  goto frommapinfo;	// skip past the default setup.
 	  }
   }
+  else
```

Comment:
Wasn't this already fixed by https://github.com/coelckers/prboom-plus/commit/dfe091e35a96e014d3f77c23b198f4a5004c6004 ?

Reply:
Looks like it given the date (6 Sep 2019) - the fix I did was some time before that but I only just thought to review changes now.

Reply:
When you said "latest branch" I thought you meant that.